index.js

```diff
@@ -69,11 +70,12 @@ function decrypt(passwd) {
  * @param     {String}    passwd      the key used to encrypt the .env into .env.enc
  * @returns   {String}                writes the encrypted file to disk at same location where the encrypted file was found and returns its md5 checksum
  */
-function encrypt(passwd) {
+function encrypt(passwd, encryptedFilename) {
```

Comment:
Variable name is from copy/pasta I guess. We are not encrypting an `encryptedFilename`. Please rename to something more meaningful like `dotEnvFilename` or something less specific.

Also add new `@param` in the function's jsdoc.